qemu/accel/tcg/cputlb.c

```diff
@@ -1577,25 +1626,6 @@ load_helper(CPUArchState *env, target_ulong addr, TCGMemOpIdx oi,
         }
     }
 
-    /* Handle CPU specific unaligned behaviour */
-    if (addr & ((1 << a_bits) - 1)) {
```

Comment:
Any reason you have to move this part forward? Just for making the `address` parameter physical address?

Reply:
This is not so good shown in the diff, I moved the `find_memory_region` and the callback handling back after the mmu lookup. Yes this is to get the physical `address`.

The unaligned check and handling is moved forward because it is done before the mmu lookup.